HPXMLtoOpenStudio/measure.rb

```diff
@@ -2760,9 +2771,14 @@ def get_space_temperature_schedule(model, location, spaces)
 
     # Sensors
     if space_values[:indoor_weight] > 0
-      sensor_ia = OpenStudio::Model::EnergyManagementSystemSensor.new(model, 'Zone Air Temperature')
-      sensor_ia.setName('cond_zone_temp')
-      sensor_ia.setKeyName(spaces[HPXML::LocationConditionedSpace].thermalZone.get.name.to_s)
+      if location == HPXML::LocationOtherHeatedSpace && !spaces[HPXML::LocationConditionedSpace].thermalZone.get.thermostatSetpointDualSetpoint.is_initialized
+        sensor_ia = UnitConversions.convert(space_values[:temp_min], 'F', 'C')
```

Comment:
This is the indoor air temperature, so here we should use the average of heating and cooling setpoints.